extract_endpoint/src/extract_endpoint/endpoint.py

```diff
@@ -29,8 +29,8 @@
 ))
 
 
-def _trigger_url() -> str:
-    return os.environ.get('TRIGGER_ADDRESS', 'http://0.0.0.0:5010') + '/run_tl'
+def _run_tl_url() -> str:
+    return os.environ.get('TL_ADDRESS', 'http://0.0.0.0:5010') + '/run_tl'
```

Comment:
Has the environment variable name been updated in Azure/Is @szinck1 aware of the change?

Reply:
He hasn't added either to Azure yet. (still need to do that and proper Mongo keys)

Reply:
He is adding the database variables as we speak